homework/shelkovich_ilya/lookUpArrayService/src/main/java/com/tutrit/java/quickstart/service/LookUpArrayService.java

```diff
@@ -0,0 +1,91 @@
+package com.tutrit.java.quickstart.service;
+
+public class LookUpArrayService {
+
+
+    public int findMax(int[] array) {
+        int check = checkIfArrayEmptyWhenSearchExtremum(array);
+        if (check < 0) {
+            return check;
+        }
+        int max = array[0];
+        for (var i = 1; i < array.length; i++) {
+            if (array[i] > max) {
+                max = array[i];
+            }
+        }
+        return max;
+    }
+
+    public int findMin(int[] array) {
+        int check = checkIfArrayEmptyWhenSearchExtremum(array);
+        if (check < 0) {
+            return check;
+        }
+        int min = array[0];
```

Comment:
change this to <=, otherwise if (check < 0)  always false

Reply:
fixed this